tests/functional/resources/test_resources.py

```diff
@@ -98,6 +98,8 @@ def test_file_upload_multiple(small_file, small_file2, uploadcare):
             for input_file in [small_file, small_file2]
         ]
     )
+    file1.close()
+    file2.close()
```

Comment:
Should be either `with open()` expression or relayed on GC, since the `assert` above could break execution

Reply:
rewrote it with contexted opens